aten/src/ATen/core/type.cpp

```diff
@@ -1544,6 +1544,11 @@ TensorTypePtr TensorType::create(const at::Tensor& t) {
   VaryingShape<size_t> stride_indices;
   VaryingShape<int64_t> strides;
   VaryingShape<int64_t> sizes;
+  // return undefined tensor type for undefined tensor
```

Comment:
This is needed, otherwise, `IValue(at::Tensor()).type()` gives a not implemented error, accessing strides of an undefined tensor.